ocs_ci/ocs/ocp.py

```
@@ -1135,16 +1135,21 @@ def wait_for_resource(
                     )
                     if status == condition:
                         log.info(
-                            f"status of {resource_name} at {column}"
-                            " reached condition!"
+                            f"status of {resource_name} at column {column}"
+                            f" reached desired condition: {condition}"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)